src/Sulu/Bundle/AdminBundle/Resources/js/containers/FieldBlocks/tests/__snapshots__/FieldBlocks.test.js.snap

```diff
@@ -10,15 +10,15 @@ exports[`Render block with schema 1`] = `
           <div class=\\"icons\\"><span class=\\"su-trash-alt clickable\\" aria-label=\\"su-trash-alt\\" role=\\"button\\" tabindex=\\"0\\"></span><span class=\\"su-times clickable\\" aria-label=\\"su-times\\" role=\\"button\\" tabindex=\\"0\\"></span></div>
         </header>
         <article>
-          <div class=\\"grid\\">
-            <div style=\\"width: 100%; margin-left: 0%; margin-right: 0%;\\" class=\\"item gridItem\\">
+          <div class=\\"grid grid\\">
```

Comment:
why we have 2 times the same class?

Reply:
It is not really the same class, one is from the `Grid` component and one from the `FieldBlocks` component.